simple_history/models.py

```diff
@@ -65,6 +65,9 @@ def __init__(self, verbose_name=None, bases=(models.Model,),
         except TypeError:
             raise TypeError("The `bases` option must be a list or a tuple.")
 
+    def _get_excluded_fields(self):
```

Comment:
Is this being used? Can't see it being called anywhere else

Reply:
It's not being used anywhere. My mistake since I hadn't spot the meta attribute containing the excluded fields in the first place.